guides/advanced-use/project-layouts.md

```diff
@@ -8,22 +8,23 @@ For example, if you have a `post` resource and a `tags` component in your projec
 ```shell
 app
 ├── components
-|   └── tags.js
+|   ├── tags.js
+|   └── tags.hbs
 ├── controllers
 |   └── post.js
 ├── models
 |   └── post.js
 ├── routes
 |   └── post.js
 ├── templates
-|   ├── components/tags.hbs
 |   └── post.hbs
-└── app.js
-└── index.html
-└── resolver.js
+├── app.js
+├── index.html
+├── resolver.js
 └── router.js
 ```
 
+Note: the default layout for non-Octane apps is to have component templates located at `app/templates/components/tags.hbs` instead of `app/components/tags.hbs` as shown above.
```

Comment:
```suggestion
Note: the default layout for non-Octane apps is to have component templates located at `app/templates/components/tags.hbs` instead of `app/components/tags.hbs` as shown above.

```